src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -35,7 +35,7 @@
     <_TargetFrameworkDirectories>$(MSBuildThisFileDirectory)/Documentation</_TargetFrameworkDirectories>
     <_FullFrameworkReferenceAssemblyPaths>$(MSBuildThisFileDirectory)/Documentation</_FullFrameworkReferenceAssemblyPaths>
     <SkipCommonResourcesIncludes>true</SkipCommonResourcesIncludes>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
latest would give you 8.0 anyway...

Reply:
Doesn't currently; complains 7.3 doesn't support the static local functions and to upgrade to 8.0 (in VS) when using latest.

Guess its "`latest` released"

Reply:
I believe `latest` is still coded for 7.3 and you have to explicitly opt into 8.0: http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/LanguageVersion.cs,480397a440092f0c,references

CC. @jaredpar

Reply:
thanks for the pointer